packages/create-sitecore-jss/src/templates/nextjs/src/lib/site-resolver/plugins/default.ts

```diff
@@ -8,6 +8,7 @@ class DefaultPlugin implements SiteResolverPlugin {
     sites.unshift({
       name: config.jssAppName,
       language: config.defaultLanguage,
+      project: config.project,
```

Comment:
Should we modify the SiteInfo type and request project field via GraphQL too? Or does that change come later?

Reply:
Valid point! It was missed, but was done in POC